java/piranha/src/test/java/com/uber/piranha/EnumConstantTest.java

```diff
@@ -33,6 +33,464 @@
 public class EnumConstantTest {
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+  /**
+   * Test for cleaning up the first enum constant in a list of enum constants.
+   *
+   * <p>See <a href="https://github.com/uber/piranha/issues/143">Issue #143</a>
+   */
+  @Test
+  public void cleanUpFirstEnumTest() {
+    ErrorProneFlags.Builder b = ErrorProneFlags.builder();
+    b.putFlag("Piranha:FlagName", "STALE_FLAG");
+    b.putFlag("Piranha:IsTreated", "false");
+    b.putFlag("Piranha:Config", "config/properties.json");
+
+    BugCheckerRefactoringTestHelper bcr =
+        BugCheckerRefactoringTestHelper.newInstance(new XPFlagCleaner(b.build()), getClass());
+
+    bcr = bcr.setArgs("-d", temporaryFolder.getRoot().getAbsolutePath());
+
+    bcr.addInputLines(
+            "TestExperimentName.java",
+            "package com.uber.piranha;",
+            "public enum TestExperimentName {",
+            " STALE_FLAG,",
```

Comment:
Added a test with a TODO, it results in invalid Java code so error-prone throws an exception (but I believe this is no different than the current behavior without the PR).

Reply:
To get some clarity on this issue, the current state of this PR for the case of an enum with a single stale flag is:
a) If the enum had no content BUT the flag enum constant then, both before and after this PR, we will end up with an empty enum which is valid Java code and follows the existing [test](https://github.com/uber/piranha/blob/master/java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java#L123).
b) Even in the case of an enum with methods/fields, it will not generate broken Java code, but just an empty constants block, terminated by `;`.
c) It will not add a comment to delete the enum currently, because that was also not the behavior of Piranha previous to this PR (i.e. we were doing it for treatment group enums but not flag enums)
Correct?